### ISSUE TITLE
"else if" does not compile

### ISSUE THREAD

**Issue by qbradley** [austral/austral#364]
As near as I can tell, using "else if" always results in a parse error.

Example program:

```
module body Test is
    pragma Unsafe_Module;

    function Put(str: FixedArray[Nat8]): Int32 is
        pragma Foreign_Import(External_Name => "puts");
    end;

    function main(): ExitCode is
        Put("Start");
        for i from 0 to 2 do
            if i = 0 then
                Put("0");
            else if i = 1 then
                Put("1");
            else
                Put("Not 0 or 1");
            end if;
        end for;
        Put("End");
        return ExitSuccess();
    end;
end module body.
```

Output:

```
Description: Austral compiler failed.
Command: ./austral compile test-programs/suites/001-trivial/014-elseif/Test.aum --entrypoint=Test:main --target-type=c --output=test-programs/output.c
Return code: 255
        --- BEGIN COMPILER STDOUT ---
        Compiler call tree printed to calltree.html

        --- END COMPILER STDOUT ---

        --- BEGIN STDERR ---
        Error:
          Title: Parse Error
          Module:
            [unknown]
          Location:
            Filename: 'test-programs/suites/001-trivial/014-elseif/Test.aum'
            From: line 18, column 12
            To: line 18, column 15
          Description:
            Error during parse.
          Code:
            16 |                 Put("Not 0 or 1");
            17 |             end if;
            18 |         end for;
            19 |         Put("End");
            20 |         return ExitSuccess();

        --- END STDERR ---

--- END ERROR ---
```

**Comment by eudoxia0** [comment]
Fixed by #366

**Comment by eudoxia0** [comment]
Thanks @ishehadeh !
